gradle/pom.gradle

```diff
@@ -1,6 +1,6 @@
 publishing.publications.all {
     //TODO github profile?
-    def devs = ["szczepiq:Szczepan Faber", "bric3:Brice Dutheil"]
+    def devs = ['szczepiq:Szczepan Faber', 'bric3:Brice Dutheil', 'raphw:Rafael Winterhalter', 'TimvdLippe:Tim van der Lippe']
```

Comment:
If you like adventures, we could add more info to every developer (for example, github, twitter, blog, etc.)